spec/views/orders/index.html.erb_spec.rb

```diff
@@ -28,7 +28,8 @@
     ])
   end
 
-  it "renders a list of orders" do
```

Comment:
Don't comment tests. If a test is broken, just add `broken: true` to it.
For example, there it will be: `it 'renders a list of orders', broken: true do`